RPIPs/RPIP-draft.md

```diff
@@ -0,0 +1,95 @@
+---
+rpip: draft
+title: oDAO Health Clarity
+description: Short and long term improvements for oDAO health clarity
+author: Valdorff (@Valdorff)
+discussions-to: https://dao.rocketpool.net/t/make-odao-health-clearer/1658/6
+status: Review
+type: Protocol
+category: Core
+created: 2023-04-17
+---
+
+
+## Abstract
+Clearer oDAO performance metrics will help oDAO members better take action to improve when
+warranted, and also allow for monitoring and accountability (from both other oDAO members and the
+community at large).
+
+## Motivation
+The current voting method for automated duties, where oDAO members stop voting once consensus is
+reached, makes it very difficult or impossible to tell if oDAO nodes are working properly based on
+performance data. This is true for the greater community, and even true for the oDAO members
+themselves.
+
+## Specification
+- In the short term, the watchtower SHALL implement a voting scheme that will show each oDAO member
+  can submit balances at least once a week
+- In the short term, the watchtower SHALL implement a voting scheme that will show each oDAO member
+  can submit prices at least once a week
+- In the medium term, the watchtower SHOULD implement a voting scheme that will show each oDAO
+  member can generate a correct merkle tree; this MAY have a moderate timeout to avoid degrading NO
+  experience much
+- The next significant smart contract release SHALL allow for voting past consensus
+
+## Reference Implementation
+### Subcommittees for balance and price duties
+This change can be implemented now in the watchtower.
+
+Each oDAO node figures out its own index (similar code already exists). On an even update
+`((current_update_block - first_update_block) / blocks_between updates) % 2 == 0`, nodes with even
+indices submitBalances right away but only submitPrices after waiting X minutes. On an odd update,
+nodes with odd indices submitBalances right away but only submitPrices after waiting 10 minutes.
+Nodes should do the work immediately and hold off on voting, to avoid unnecessary delay when
+submitting after their 10-minute wait. Note: consensus currently requires more than half of votes,
+so this approach will often mean we need to hit the 10-minute wait before consensus is possible.
+
+### Subcommittee for reward snapshot duty
+This change can be implemented now; however, trees currently take a long time and the subcommittee
+approach works using generous buffers. As a result, it's recommended that this change be implemented
+after trees are reliably fast enough in the watchtower such that this approach doesn't degrade NO
+experience significantly.
+
+Each oDAO node figures out its own index (similar code already exists). On an even rewards period,
+nodes with even indices submitBalances right away but only submitPrices after waiting 30 minutes.
+On an odd update, nodes with odd indices submitBalances right away but only submitPrices after
```

Comment:
This should say "nodes with odd indices submitRewardSnapshot right away and nodes with even indices should only submit after waiting 30 minutes"